### ISSUE TITLE
Polls command

### ISSUE THREAD

**Issue by Arnav-2004** [gurkult/gurkbot#75]
How about a polls command which can be used to create polls? I don't know how useful this will.
![image](https://user-images.githubusercontent.com/70969910/104814585-c9f94680-5835-11eb-8017-78a3f98f85f1.png)
It could look something like this

**Comment by ReneganRonin** [comment]
I would be very glad if someone can implement this. @gustavwilliam i think it is a good idea?

**Comment by gustavwilliam** [comment]
Yeah, this would be nice. Don’t we already have an open issue for it though?

**Comment by Arnav-2004** [comment]
> Yeah, this would be nice. Don’t we already have an open issue for it though?

No I don't think so

**Comment by Shivansh-007** [comment]
We can close this since, it is no longer needed. Thanks! 
-According to the discussion in [#dev-gurkbot](https://discord.com/channels/789192517375623228/789295038315495455/803528818160697366).

**Comment by onerandomusername** [comment]
Might try to implement this if still wanted

**Comment by gustavwilliam** [comment]
Sure. Would be nice to have.

Allow anyone to use the command. And don't copy the PyDis code for this, so a too large portion of our codebase won't be PyDis-based.

**Comment by TheLegendBeacon** [comment]
I have a working implementation elsewhere, I could probably copy my code over to here.

**Comment by onerandomusername** [comment]
I'll probably use interactions for this,  as that would (a)  look nice and (b)  not require reaction add listeners

**Comment by gustavwilliam** [comment]
How would that work? Is the embed updated to display the current stats?

**Comment by onerandomusername** [comment]
Yeah, would edit on a button press. 

I suppose the small problem with interactions is that its not easily possible to tell at a glance which you voted for.... hm.
